README.md

```diff
@@ -80,8 +93,16 @@ Using jQuery, add an event listener for clicks on the button with the id
 body that says "hello".
 
 Your Answer:
-```js
-// your code here
+```
+$(document).ready(function(){
+  $("button").click(function(){
+      $("p").append("hello");
+  });
+}
+<p>Greeting</p>
+<button id="button">Hello</button>
+//
```

Comment:
Close.  Please reread the questions, looking at the specific elements we are expecting.